m4/pdns_check_os.m4

```diff
@@ -2,7 +2,7 @@ AC_DEFUN([PDNS_CHECK_OS],[
   THREADFLAGS=""
 
   case "$host_os" in
-  solaris2.10)
+  solaris2.10 | solaris2.11)
```

Comment:
That is far less likely than 2.12 being exactly the same, in which case we need to edit. It is a gamble either way, but I like gambles that have a fair chance of not causing action :)